File: Python-Basics/loops/salary.py
n = int(input())
salary = float(input())

for i in range(1,n):
    website = input()
    if website == 'Facebook':
        salary -= 150
    elif website == 'Instagram':
        salary -= 100
    elif website == "Reddit":
        salary -= 50
    if salary == 0:
        print('You have lost your salary')
        break

if salary != 0:
    print(f'{salary:.0f}')

File: CPP-Basics/Salary.cpp
#include <iostream>
#include <string>

using namespace std;

int main () {

    int n;
    cin >> n;

    double salary;
    cin >> salary;

    for(int i = 0; i< n; i++) {

        string site;
        cin >> site;

        if(site == "Facebook") {

            salary -= 150;

        } else if(site == "Instagram") {

            salary -= 100;

        } else if(site == "Reddit") {

            salary -= 50;
        }

        if(salary <= 0) {

        cout << "You have lost your salary." << endl;
        break;

    }
    }

    if(salary > 0){

        cout << salary << endl;
    }

    return 0;
}
